app/views/bookings/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for @booking do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name, value: @movie.name, disabled: true ,class: "form-control" %>
+  <%= f.label :date %>
+  <%= f.text_field :date,value: @schedule.start_at, disabled: true , class: "form-control" %>
+  <%= f.label :email %>
+  <%= f.text_field :email, value: current_user.email, disabled: true , class: "form-control" %>
+  <%= f.label :phone %>
+  <%= f.text_field :phone,value: current_user.phone, disabled: true , class: "form-control" %>
+  <div class="gender">
+    <%= f.radio_button :payment_id, 1, :checked => true %>
+    <%= f.label :counter %>
+    <%= f.radio_button :payment_id, 2 %>
```

Comment:
Sao lại có giá trị 1, 2 ri hì? giải thích nghiệp vụ chỗ này giúp chị?
reply hoặc lên bàn chị nói trực tiếp..tùy cậu

Reply:
Dạ chỗ này là phương thức thanh toán , tại quầy hoặc paypal, 1 và 2 là id của 2 phương thức đó. Nên khi người dùng chọn phương thức nào thì sẽ tạo 1 booking với id của payment method

Reply:
giá trị 1, 2 thì khai báo vào setting nhé, cơ mà nếu rứa thì em tìm hiểu về enum cho payment_type (payment_id nên là payment_type) seẽ hợp lý hơn

Reply:
Em mới update lại

Reply:
Da em moi update lai